samples/gen-go-deprecated/server/interface.go

```diff
@@ -11,6 +11,12 @@ import (
 // Controller defines the interface for the swagger-test service.
 type Controller interface {
 
-	// GetBook makes a GET request to /books/{id}.
+	// GetBook makes a GET request to /books/{id}
+
+	// 200:
```

Comment:
maybe this should read `200: nil`?

also kind of weird that this method doesn't return a book...